docs/deploy-development-rp.md

```diff
@@ -76,42 +76,109 @@
+    1. Confirm the nmap output looks like this: (if it does not then your VPN is not connected correctly; kill anything using port 443 and connect again)
+    ```bash
+    Starting Nmap 7.92 ( https://nmap.org ) at 2022-03-29 18:29 EDT
+    Nmap scan report for 10.0.0.4
+    Host is up (0.015s latency).
+
+    PORT STATE SERVICE
+    443/tcp open https
+
+    Nmap done: 1 IP address (1 host up) scanned in 0.25 seconds
+    ```
+    1. Update the PROXY_HOSTNAME environment variable in ./secrets/env to point the IP you located above of for the Proxy VM.
+  1. Now that your VPN is connected correctly and you've updated PROXY_HOSTNAME you need to source your env file for that update.
+  ```bash
+  . ./secrets/env
+  ```
+  1. Execute the local ARO-RP
+  ```bash
+  make runlocal-rp
+  ```
 
-## Run the RP and create a cluster
-
-1. Source your environment file.
-
-   ```bash
-   . ./env
-   ```
-
-1. Run the RP
+## Steps to Create Cluster:
 
-   ```bash
-   make runlocal-rp
-   ```
+  1. Open another terminal (make sure you source your ./secrets/env file in this terminal as well)
+  1. Execute this command to create a cluster
+  ```bash
+  CLUSTER=<aro-cluster-name> go run ./hack/cluster create
+  ```
 
-1. To create a cluster, EITHER follow the instructions in [Create, access, and
-   manage an Azure Red Hat OpenShift 4.3 Cluster][1].  Note that as long as the
-   `RP_MODE` environment variable is set to `development`, the `az aro` client
-   will connect to your local RP.
+  This will take a while but eventually if the cluster is created you should see the following in your terminal indicating the cluster creation was successful:
+  ```bash
+  INFO[2022-04-01T10:02:41-05:00]pkg/util/cluster/cluster.go:318 cluster.(*Cluster).Create() creating cluster complete
+  ```
 
-   OR use the create utility:
+## Steps to connect to the Cluster and confirm it is up via kubectl or oc:
 
+1. At your terminal execute to create the admin.kubeconfig locally. This will allow you to connect to the cluster via kubectl or oc
    ```bash
-   CLUSTER=cluster go run ./hack/cluster create
+   CLUSTER=<aro-cluster-name> make admin.kubeconfig
    ```
 
-   Later the cluster can be deleted as follows:
+1. Disconnect from "rp-vnet" vpn and connect to "dev-vnet" vpn. The steps are identical to connecting to "rp-vnet" in the #preparation-to-create-cluster section. You will just need to download the dev-vpn client certificate locally, create the VPN connection using your VPN client of choice, and use nmap to get the IP of the internal load balancer from the <aro-cluster-name-rg>. You can find this address at: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/<subscriptionid>/resourceGroups/<aro-cluster-rp-rg>/providers/Microsoft.Network/loadBalancers/<aro-cluster-name>-<random string for your lb>-internal/frontendIpPool (internal-lb-ip-v4). In my case the IP was 10.62.174.
```

Comment:
Would it be easier to include `az` commands rather than links to portal?

Reply:
@cadenmarchese I think that is a nifty idea but I would rather do it as a separate PR if that sounds good to you. I'll be glad to do that for all of the documented we've worked on recently if need be. For the moment I adjusted this to backticks to follow suit with our other example urls.